### ISSUE TITLE
Blog posts - Mozfest website

### ISSUE THREAD

**Issue by Saallen** [mozilla/2015.mozillafestival.org#254]
Issue to curate Mozfest blog page 
List of blogs to be published
- [x] Localisation 
- [x] Badges - how to orientate yourself at the festival through pathways
- [x] Bekka Kahn Learning Circles
- [ ] How to be a good participant and get the most of the festival
- [x]  iFLabs
- [x] Digital Citizenship
- [x] Volunteers

I don't have these posts yet but should have the first by the middle of next week
cc @hannahkane  cc @KevZawacki

**Comment by hannahkane** [comment]
We don't have a MozFest blog, as far as I know. Is this ticket for creating one? Or will these blogs be published on the Webmaker blog and then displayed or linked to from the MozFest site?

**Comment by Saallen** [comment]
......checked other issue and ..". We can also make a separate "page" for these posts on blog.webmaker.org, so they have a dedicated space. For example, blog.webmaker.org/MozFest2015. "
So additional page on webmaker. I could always just tweet the links to the posts instead of adding them to the website

**Comment by KevZawacki** [comment]
Hi @Saallen, just to confirm -- you'll send drafts of these blog my way, and I'll proofread and post. Sound right?

**Comment by hannahkane** [comment]
@Saallen - we can add this content to the website. Was just trying to clarify if that was the request. If so, let's file a separate ticket and add specific details about how/where we'd like this to appear on the site.

**Comment by Saallen** [comment]
Hey I worry though if I only get 4 blog posts is it worth being on the website?

@KevZawacki , yes proof only :) Unless you would like to write one about the main stage speakers? :)

**Comment by KevZawacki** [comment]
@Saallen I am happy to help, but probably cannot commit fully -- also juggling the State of Mozilla report. Are you able to put together a rough draft? I can hone it from there.

**Comment by Saallen** [comment]
First post from Badges 
https://docs.google.com/document/d/1qoftVkt9i1DEH1DQMDgpgSo55NzswAGUhZtz1G_oVAg/edit?ts=5617d7c8
@KevZawacki I realise you are extemely busy, I am asking everyone for a Blog post :)

**Comment by hannahkane** [comment]
Got it re: not putting on website. :)

**Comment by KevZawacki** [comment]
Hi @Saallen, just gave the [badges blog](https://docs.google.com/document/d/1qoftVkt9i1DEH1DQMDgpgSo55NzswAGUhZtz1G_oVAg/edit?ts=5618b17d) a copyedit. Ready for me to post it to [blog.mozilla.org/faces-of-mozilla](blog.mozilla.org/faces-of-mozilla)?

And no problem — I will put together a main stage blog post once we finalize the details.

**Comment by Saallen** [comment]
@KevZawacki I just shared the page with edits with the badges team so once i hear from them we are good to go.

**Comment by Saallen** [comment]
@KevZawacki Badges team are happy for you to proceed \0/

**Comment by KevZawacki** [comment]
Great! Our first MozFest blog is live [here](http://blog.webmaker.org/mozfest-2015). @Saallen, let me know if I should update the description at the top of the page.

**Comment by Saallen** [comment]
Yes please @KevZawacki - can we update the intro paragraph
Proposed content:
MozFest 2015

Welcome to the Mozilla Festival 2015 blog page. Here, you’ll find posts from this years Space Wranglers, Pathfinder and volunteers who make the festival come to life. Over the coming weeks you will be introduced the over arching Spaces which are our central hubs of learning and you will learn how to complete a pathways with a focused skill set. We will introduce you to the teams, the festival narratiev and how to get the best out of the festival weekend

MozFest is scheduled for Nov. 6-8 in London. For additional details head to
https://2015.mozillafestival.org/

**Comment by Saallen** [comment]
@KevZawacki Localisation just submitted their blog post
[Localisation Blog post.docx](https://github.com/mozilla/2015.mozillafestival.org/files/13146/Localisation.Blog.post.docx)
![super heros](https://cloud.githubusercontent.com/assets/10194015/10482565/f8ebee62-7270-11e5-9f62-ebd4e79b3365.png)

**Comment by Saallen** [comment]
[MozFest IF!LABS blog-1.docx](https://github.com/mozilla/2015.mozillafestival.org/files/15944/MozFest.IF.LABS.blog-1.docx)
imap://sarah%40mozillafoundation%2Eorg@imap.googlemail.com:993/fetch%3EUID%3E/INBOX%3E24100?part=1.2.2&filename=11947478_870584649696972_1133626732403078432_n.jpeg

**Comment by SpikeUK** [comment]
For information only. MozFest volunteer Ailsa Abraham is putting together a blog about the event from the vols POV. Getting input from two other volunteers as well. Should be available in next two days.

**Comment by KevZawacki** [comment]
@Saallen Thanks! Our [blog page](http://blog.webmaker.org/mozfest-2015) is now up to date.

**Comment by SpikeUK** [comment]
Blog info  from Ailsa Abraham (mentioned two comments up) is here >> https://docs.google.com/document/d/1UsrI3ysxfZnk2WUiHq7RaKBx4ZvNePSQ6OkGrVHGAgU/edit?usp=sharing

**Comment by Saallen** [comment]
Thanks @SpikeUK 
@KevZawacki are we able to add pictures to this blog page as it looks like  alot of dense text?

**Comment by Saallen** [comment]
or @KevZawacki give @marcwalsh the log on details and he can do this

**Comment by KevZawacki** [comment]
@Saallen @SpikeUK Ailsa's blog posted, and photos added to previous blogs. Thanks!

@Saallen, we're also planning on publishing these blogs on the MozFest Medium publication. OK by you?

**Comment by Saallen** [comment]
:+1:

**Comment by Saallen** [comment]
Adding Digital Citizenships blog post 
https://docs.google.com/document/d/1QlwR7t9nldzrdWXv8TlQMf0uNLKcyKzhV3_ao7xutPw/edit
@Melechuga can you add a picture for the page?

**Comment by Saallen** [comment]
@KevZawacki the layout of the page is hard to follow now we have so many posts- is there anything  we can do to help this?

**Comment by SpikeUK** [comment]
Another pic please to go in Ailsa's Volunteer blog. Nice Paul Clarke picture of David Ross (CC BY-SA 2.0) here >> https://www.flickr.com/photos/paul_clarke/15447640209/in/album-72157648560922709/
Thank you

**Comment by KevZawacki** [comment]
@Saallen Digital Citizenship blog copyedited and published. I've also added @SpikeUK's photo to Ailsa's blog.

Unfortunately, WordPress won't let us do any complex formatting. :-(

**Comment by SpikeUK** [comment]
@KevZawacki Thank you! Looks wonderful :)

**Comment by Saallen** [comment]
hi @KevZawacki 
Arts award have shared us their blog post for the festival .https://docs.google.com/document/d/1aqbIZVF_123ft9emdsor2tK5RXXKHJTARzPSyyKkSns/edit
![artsawardrgb](https://cloud.githubusercontent.com/assets/10194015/10786584/e8f8486e-7d62-11e5-990b-a90d4e1de594.jpg)

**Comment by cassiemc** [comment]
@Saallen update?

**Comment by KevZawacki** [comment]
@Saallen Arts Award blog added!

**Comment by Saallen** [comment]
Hi @cassiemc - what update do you need from me?

**Comment by Saallen** [comment]
Second last blog post  From Beka Khann - cc @KevZawacki 
[mozfest15_blogpost-1.docx](https://github.com/mozilla/2015.mozillafestival.org/files/24157/mozfest15_blogpost-1.docx)

**Comment by KevZawacki** [comment]
Thanks, @Saallen -- just published.

**Comment by Saallen** [comment]
:banana:
